src/amuse/support/options.py

```diff
@@ -90,137 +90,135 @@ def homedirectory(self):
                     path = os.environ[evar]
                     if os.path.isdir(path):
                         break
-                except: pass
+                except:
+                    pass
         if path:
             return path
         else:
             raise RuntimeError('please define environment variable $HOME')
-            
+
     @classmethod
-    def instance(cls, preloadfp = None):
+    def instance(cls, preloadfp=None):
         if cls.INSTANCE is None:
             cls.INSTANCE = cls()
             cls.INSTANCE.load(preloadfp)
         return cls.INSTANCE
-        
+
     def get_value_for_option(self, option, instance):
         if option.name in self.overriden_options:
             return self.overriden_options[option.name]
-            
+
         for x in option.get_sections(instance):
             if self.config.has_option(x, option.name):
                 return option.get_value(instance, x, self.config)
         return option.get_defaultvalue(instance)
-        
 
     def to_ini_string(self):
         file = StringIO()
         self.config.write(file)
         return file.getvalue()
-    
-    
+
     def override_value_for_option(self, name, value):
         self.overriden_options[name] = value
-        
 
     def read_from_ini_string(self, string):
         file = StringIO(string)
-        self.config.readfp(file)
-    
-    
+        self.config.read_file(file)
```

Comment:
confirm this is ok?

Reply:
yes - again a deprecation warning with specific fix listed